checker/src/org/checkerframework/checker/index/upperbound/OffsetEquation.java

```diff
@@ -387,7 +387,7 @@ private static int parseInt(String intLiteral) {
         if (intLiteral.isEmpty()) {
             return 0;
         }
-        return Integer.valueOf(intLiteral);
+        return Long.valueOf(intLiteral).intValue();
```

Comment:
>The reason I changed this line is that the Value Checker sometimes reports that an offset is an "integer" that's outside of the int range, but OffsetEquation's isInt method will still return true. 

Do you have a test case that shows this problem?

>I could replace this change with a change in isInt that checks the size of the integer, instead.

I think that changing isInt makes more sense, because if it's too long, it's not an int.